pkg/forwarder/transaction/transaction.go

```diff
@@ -384,3 +384,18 @@ func (t *HTTPTransaction) SerializeTo(serializer TransactionsSerializer) error {
 	log.Trace("The transaction is not stored on disk because `storableOnDisk` is false.")
 	return nil
 }
+
+// readResponseBody read HTTP response body. If body is more than 1000 bytes
+// truncates it to  prevent from logging a huge message.
+// 1000 bytes are enough to tell which tools a response comes.
+func readResponseBody(body io.ReadCloser) ([]byte, error) {
+	b, err := io.ReadAll(body)
```

Comment:
If we're worried about reading too much data in the response, I think [io.ReadFull](https://pkg.go.dev/io#example-ReadFull) may be a better option since we know the max size of the buffer.